nautobot_chatops/api/views/mattermost.py

```diff
@@ -36,7 +37,7 @@ def verify_signature(request):
         # in these cases, we have to load the request.body
         try:
             data = json.loads(request.body)
-        except ValueError as e:
+        except ValueError as e:  # pylint:disable=invalid-name
```

Comment:
It'd be better to just change `e` to something like `exc` throughout, as that would stop pylint from complaining.

Reply:
Sure,will do.